portfolio/pom.xml

```diff
@@ -49,6 +49,11 @@
       <artifactId>jsr305</artifactId>
       <version>3.0.2</version>
     </dependency>
+    <dependency>
+  	  <groupId>com.google.cloud</groupId>
```

Comment:
nit: 2 spaces :)

Reply:
ahh I don't know why it's not showing up on cloudshell the same way it's showing on github -- will try to figure that out!